src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
-                                <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
-                                  {dashboard?.view_count}
-                                </Text>
-                              </Layout.Horizontal>
-
-                              <Layout.Horizontal>
-                                <Icon name="star-empty" size={18} style={{ marginRight: 'var(--spacing-4)' }} />
-                                <Text color={Color.BLACK_100} font={{ size: 'normal', weight: 'semi-bold' }}>
-                                  {dashboard?.favorite_count}
-                                </Text>
-                              </Layout.Horizontal>
-                            </>
-                          )}
-                        </Layout.Horizontal>
-                      </Layout.Vertical>
-                    </Container>
-                  </Card>
-                )}
-                keyOf={dashboard => dashboard?.id}
-              />
-            </Container>
-          )}
+                        {tag}
+                      </section>
+                    )
+                  }
+                })}
 
-          {filteredDashboardList && filteredDashboardList.length > 0 && layoutView === LayoutViews.LIST && (
-            <Container className={css.masonry}>
-              <TableV2<DashboardInterface> className={css.table} columns={columns} data={filteredDashboardList || []} />
+              {tagsList?.resource?.tags?.length === 0 && <span>{getString('dashboards.homePage.noTags')}</span>}
             </Container>
-          )}
+          </Layout.Vertical>
+        </Container>
+      </Layout.Vertical>
 
-          {filteredDashboardList && filteredDashboardList.length === 0 && !loading && (
-            <Container style={{ height: 'calc(100vh - 226px)' }} flex={{ align: 'center-center' }}>
-              <Layout.Vertical spacing="medium" width={470} style={{ alignItems: 'center', marginTop: '-48px' }}>
-                <Icon name="dashboard" color={Color.GREY_300} size={35} />
-                <Heading level={2} font={{ align: 'center' }} color={Color.GREY_500}>
-                  {getString('dashboards.homePage.noDashboardsAvailable')}
-                </Heading>
-              </Layout.Vertical>
-            </Container>
+      <Layout.Vertical
+        padding="large"
+        style={{ width: 'calc(100% - 280px)', height: 'calc(100vh - 226px)', paddingTop: 0, overflow: 'scroll' }}
+      >
+        <Layout.Horizontal style={{ padding: 'var(--spacing-6) var(--spacing-9) ' }}>
+          <ExpandingSearchInput
+            placeholder={getString('dashboards.homePage.searchPlaceholder')}
+            onChange={(text: string) => {
+              setSearchTerm(text)
+            }}
+            className={css.search}
+          />
+        </Layout.Horizontal>
+        <Layout.Horizontal style={{ marginLeft: 'var(--spacing-xxxlarge)', alignItems: 'baseline' }}>
```

Comment:
You can use the `margin` and `flex` props or extract to a class
```suggestion
        <Layout.Horizontal margin={{ left: 'xxxlarge' }} flex={{ alignItems: 'baseline' }}>
```